indico/modules/events/registration/views.py

```diff
@@ -24,7 +24,18 @@
 
 class WPManageRegistration(WPJinjaMixin, WPConferenceModifBase):
     template_prefix = 'events/registration/'
-    sidemenu_option = 'registration'
+
+    @property
+    def sidemenu_option(self):
+        if self._conf.getType() != 'conference':
```

Comment:
why not just return registration if it's a conference? less indentation for the following code!